backend/e4h-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -171,6 +171,41 @@ async def get_facility_ingestion_template_with_data(
+        # Create an empty DataFrame with the expected boundary columns
+        df = pd.DataFrame(columns=["Country", "State", "District", "Block"])
+        df.to_excel(output_file_path, sheet_name="Boundary Data", index=False)
+
+        logger.info(f"Successfully created boundary ingestion template at {output_file_path}")
+
+        return FileResponse(
+            path=output_file_path,
+            filename=output_filename,
+            media_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)